### DISCUSSION TITLE
Fenced Code Block

### DISCUSSION THREAD

**Discussion by palokl** [gsantner/markor#1523] [Q&A] [answered]
Markdown code block is supported in Edit mode by indenting lines by 4 spaces - this is working nice - code block is gray
But fenced code block, by using three backticks (```) before and after the code block is not working, block is not highlighted. Would be possible implement it ?

In Viewing mode would be possible to show code block in gray area or by color font ? monospace font used for code block in View mode is indistinct.

Thanks for great Markdown editor !

**Comment by gsantner** [comment, marked as answer]
The backticks block is performance-wise currently tricky, so we don't have it.  
If there is somebody who can make it - without a major performance degrade - I'm happy to add it to Markor.

View mode and Edit mode are complete separate things, one is basically a browser, the other a normal text editor. The used fonts and their styles are similar, but not the same.
